### DISCUSSION TITLE
trying to train  Efficient net

### DISCUSSION THREAD

**Discussion by acharjee07** [Lightning-AI/pytorch-lightning#7465] [code help: CV] [answered]
But I keep getting this error. I have no what it's about. A hint will be really helpful. Thanks in advance.

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-147-594f245450d7> in <module>
----> 1 trainer.fit(LitPlant, train_dataloader=train_loader, val_dataloaders=valid_loader)

/opt/conda/lib/python3.7/site-packages/pytorch_lightning/trainer/trainer.py in fit(self, model, train_dataloader, val_dataloaders, datamodule)
    447         # ----------------------------
    448         # setup data, etc...
--> 449         self.train_loop.setup_fit(model, train_dataloader, val_dataloaders, datamodule)
    450 
    451         # hook

/opt/conda/lib/python3.7/site-packages/pytorch_lightning/trainer/training_loop.py in setup_fit(self, model, train_dataloader, val_dataloaders, datamodule)
    112         # clean hparams
    113         if hasattr(model, "hparams"):
--> 114             parsing.clean_namespace(model.hparams)
    115 
    116         # links data to the trainer

/opt/conda/lib/python3.7/site-packages/pytorch_lightning/utilities/parsing.py in clean_namespace(hparams)
     73         hparams_dict = hparams.__dict__
     74 
---> 75     del_attrs = [k for k, v in hparams_dict.items() if not is_picklable(v)]
     76 
     77     for k in del_attrs:

AttributeError: 'property' object has no attribute 'items'

**Comment by ananthsub** [comment, marked as answer]
What's the type of LitPlant? Are you passing an instance of the class or the class type to trainer.fit() ?

**Reply by acharjee07** [reply]
The model just started training !!!! I accidentally passed the class type instead of an instance. Thank you so much for pointing to the mistake.
